src/components/UserDashboard/AuthorizedStudies/Study.js

```diff
@@ -33,12 +33,14 @@ const Codes = styled(StudyCount)`
   margin-bottom: 5px;
 `;
 
+// isn't this an internal link?
```

Comment:
@bdolly correct. bad semantic use. will fix in different branch, let's leave comments like this to a different channel (slack/email) so it doesn't set a precedent for communication

Reply:
@ciaranschutte  understood